webpack.common.js

```diff
@@ -36,13 +78,31 @@ config = [
                 {
                     test: /\.css$/,
                     loader: ExtractTextPlugin.extract(
-                        combineLoaders([{
+                        combineLoaders([
+                          {
                             loader: 'css-loader',
                             query: {
                                 modules: true,
                                 localIdentName: '[local]'
+                            },
+                            //can't use minimize because cssnano (the dependency)
+                            //doesn't parser the browserlist extension in packag.json
+                            //correctly, there's a pending update for it but
+                            //css-loader doesn't have the latest yet.
+                            // options: {
+                            //   minimize: true
```

Comment:
I spent a bit of time on this because I really wanted to minimize our css on production builds.  However it can't be done yet because of some dependency conflicts between the `cssnano` hasn't been updated yet to use the latest `browserslist` version which supports the `extends` syntax within `package.json`.  So just left this comment so when `cssnano` is updated to version 4 (currently its a release candidate) we'll be able to use it.  

It's possible at that point I may need to update the webpack config to webpack4 as well (which was just released recently).  But that still has a few quirks with not all packages supporting webpack 4 yet so I'm holding off for the time being.